### PR TITLE
IGVF-107 treatment updates

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

**Review by cricketsloan — CHANGES_REQUESTED** [review]
There was one spelling error, but that lead to me trying to tidy up the sentences overall.

File: src/igvfd/schemas/treatment.json [hunk 1]
```diff
@@ -100,7 +106,8 @@
         "amount": {
             "title": "Amount",
             "type": "number",
-            "description": "Amount in conjunction with amount_units is used to indicate the exact quantity of the specific treatmenat that was used."
+            "description": "Amount in conjunction with amount_units is used to indicate the exact quantity of the specific treatmenat that was used.",
```
[the remark this answers is not in the document]
Comment (gabdank) [reply src/igvfd/schemas/treatment.json:109]: done

File: src/igvfd/schemas/treatment.json [hunk 2]
```diff
@@ -120,20 +127,21 @@
             "comment": "amount_units should be used in conjunction with amount."
         },
         "duration": {
-            "title": "Time duration",
+            "title": "Duration",
             "type": "number",
-            "description": "Duration in conjunction with duration_units is used to indicate the time that elapsed between start of the treatment upto the end of the treatment."
+            "description": "Duration is used to indicate the time that elapsed between start of the treatment upto the end of the treatment.",
```
[the remark this answers is not in the document]
Comment (gabdank) [reply src/igvfd/schemas/treatment.json:132]: done

File: src/igvfd/schemas/treatment.json [hunk 3]
```diff
@@ -100,7 +106,8 @@
         "amount": {
             "title": "Amount",
             "type": "number",
-            "description": "Amount in conjunction with amount_units is used to indicate the exact quantity of the specific treatmenat that was used."
+            "description": "Amount in conjunction with amount_units is used to indicate the exact quantity of the specific treatmenat that was used.",
+            "comment": "amount should be used in conjunction with amount_units."
```
[the remark this answers is not in the document]
Comment (gabdank) [reply src/igvfd/schemas/treatment.json:110]: done

File: src/igvfd/schemas/treatment.json [hunk 4]
```diff
@@ -120,20 +127,21 @@
...
+            "title": "Duration units",
             "type": "string",
             "enum": [
                 "second",
                 "minute",
                 "hour",
                 "day"
             ],
-            "comment": "duration_units should be used in conjunction with duration"
+            "comment": "duration_units should be used in conjunction with duration."
```
[the remark this answers is not in the document]
Comment (gabdank) [reply src/igvfd/schemas/treatment.json:144]: done

**Review by cricketsloan — APPROVED** [review]

**Pull request merged**